### DISCUSSION TITLE
need help?

### DISCUSSION THREAD

**Discussion by sabahtalateh** [madwareru/intellij-ron-remix#17] [General]
Hello, in CLion 2020.3 go to definition and completion not working. i can try to help on this but don't know how to write plugins for IntelliJ yet. can you give some overview of the project or i can just walk through the https://github.com/JetBrains/intellij-platform-plugin-template?

my telegram is @grogenverg

**Comment by madwareru** [comment]
Hi Alex. At first we need to be sure which kind of go to definition you want. Do you want to be able to map Ron files on Rust types? I'd be glad to guide you on the project structure anyway. And you should definitely check out this doc: https://plugins.jetbrains.com/docs/intellij/
